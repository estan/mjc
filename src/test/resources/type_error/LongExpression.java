class LongExpressionTest {
    public static void main(String[] args) {
        long l;
        //l = 9223372036854775807l; // OK!
        //l = 9223372036854775807L; // OK!
        l = 9223372036854775808l; // INVALID_LONG_LITERAL
        l = 9223372036854775808L; // INVALID_LONG_LITERAL
    }
}
