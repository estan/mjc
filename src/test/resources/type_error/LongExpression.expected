INVALID_LONG_LITERAL
INVALID_LONG_LITERAL
